volplugin/init.go

```diff
@@ -58,12 +59,16 @@ func (dc *DaemonConfig) updateMounts() error {
 			log.Infof("Refreshing existing mount for %q", mount.Volume.Name)
 
 			vol, err := dc.requestVolume(parts[0], parts[1])
-			switch err {
-			case errVolumeNotFound:
-				log.Warnf("Volume %q not found in database, skipping")
-				continue
-			case errVolumeResponse:
-				log.Fatalf("Volmaster could not be contacted; aborting volplugin.")
+			if erd, ok := err.(*errored.Error); ok {
+				switch {
+				case erd.Contains(errors.NotExists):
+					log.Warnf("Volume %q not found in database, skipping")
+					continue
+				case erd.Contains(errors.GetVolume):
+					log.Fatalf("Volmaster could not be contacted; aborting volplugin.")
```

Comment:
this is minor and probably not in the scope of this PR but thought will share and see if we need to handle this.

Just thinking out loud, for cases like these should we just wait and retry like we do for etcd as volmaster might not be up when volplugin starts